pallets/valve/src/mock.rs

```diff
@@ -267,38 +87,35 @@ impl Contains<Call> for ClosedCallFilter {
 	}
 }
 
-pub struct MockWeight<T>(PhantomData<T>);
-impl<Test: frame_system::Config> pallet_valve::WeightInfo for MockWeight<Test> {
-	fn close_valve() -> Weight {
-		0
-	}
-	fn open_valve() -> Weight {
-		0
-	}
-	fn close_pallet_gate_new() -> Weight {
-		0
-	}
-	fn close_pallet_gate_existing() -> Weight {
-		0
-	}
-	fn open_pallet_gate() -> Weight {
-		0
-	}
-	fn open_pallet_gates() -> Weight {
-		0
+thread_local! {
+	pub static MOCK_AUTOMATION_TIME_STORE: RefCell<bool> = RefCell::new(false);
```

Comment:
I picked up this pattern for mocking in tests from the Session pallet. Big improvement over having to stub the entire pallet.